torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -222,14 +222,14 @@ class ProcessGroup : public torch::CustomClassHolder {
   // Gathers a single tensor inputBuffer into a single buffer outputBuffer that
   // is interpreted as a contigious collection of size inputBuffer * WORLD_SIZE.
   // For implementers of ProcessGroup API and advanced users only.
```

Comment:
Could you update the comment saying this function will be deprecated going forward?